src/test/test_utils.hpp

```diff
@@ -128,6 +130,8 @@ constexpr const float INCREMENT_1x3x4x5_ADDITION_VALUE = 1.0;
 
 constexpr const ovms::model_version_t UNUSED_MODEL_VERSION = 42;  // Answer to the Ultimate Question of Life
 
+static const ovms::ExecutionContext DEFAULT_CONTEXT{ovms::ExecutionContext::Interface::GRPC, ovms::ExecutionContext::Method::Predict};
```

Comment:
maybe rename to DEFAULT_TEST_CONTEXT?

Reply:
To be added in next metric PR

Reply:
Resolved in: https://github.com/openvinotoolkit/model_server/pull/1389